admin_guide/out_of_resource_handling.adoc

```diff
@@ -87,13 +90,13 @@ If not specified, the node kills pods immediately with no graceful termination.
 
 To configure soft eviction thresholds, the following flags are supported:
```

Comment:
With the below, I think an example would be a lot better... @derekwaynecarr Do you have one we could put into the docs?